admin-api.yaml

```diff
@@ -227,6 +227,160 @@ paths:
                 $ref: '#/components/schemas/Project'
       security:
         - basicAuth: []
+  /projects/{project_id}/delivery-usage:
+    get:
+      tags:
+        - Usage
+      summary: Get delivery usage
+      description: This endpoint enables you to retrieve the total amount of minutes delivered for a specific project based on its `project_id`.
+      parameters:
+        - in: path
+          name: project_id
+          required: true
+          schema:
+            type: string
+          description: The ID of the project you want to retrieve usage data for.
+        - in: query
+          name: interval
+          required: true
+          schema:
+            type: string
+            enum:
+              - month
+              - week
+          description: Set the period of time that you want to retrieve delivery usage for. `month` returns data for the past 30 days, while `week` returns data for the past 7 days.
+      responses:
+        '200':
+          description: Successful response
+          content: 
+            application/json:
+              schema:
+                type: object
+                properties:
+                  '{date}':
+                    description: A date within the interval you selected in `YYYY-MM-DD` format.
+                    type: object
+                    pattern: '^\d{4}-\d{2}-\d{2}$'
```

Comment:
I added pattern because it does not break the OpenAPI validation, even though it is skipped. 🤷